react/index.d.ts

```diff
@@ -2610,9 +2610,7 @@ declare namespace React {
         isRequired: Validator<T>;
     }
 
-    interface ValidationMap<T> {
-        [key: string]: Validator<T>;
-    }
+    type ValidationMap<T> = { [K in keyof T]?: Validator<T> };
```

Comment:
Is the value type really `Validator<T>` and now `Validator<T[K]>`? Not familiar with the library but the latter is more common.